lighttag_to_adm.py

```diff
@@ -5,12 +5,26 @@
 from type_definitions.lighttag_job import JobResult
 from type_definitions.lighttag_result_types import Example, Annotation
 from utils import sort_example_annotations, make_slug
+from configuration import ConfigurationUtil
 import json
 import os
 
+
+config = ConfigurationUtil.get_configuration()
```

Comment:
This should be done in the main function and passed as a parameter.
As it stands the module can't be tested without a config